helm/ph-ee-engine/templates/tests/_helpers.tpl

```diff
@@ -45,6 +45,8 @@ It directly provides values.
           fieldPath: status.podIP
     - name: "CALLBACK_URL"
       value: "http://$(MY_POD_IP):53013"
+    - name: "EMAIL_URL"
+      value: "http://$(MY_POD_IP):3025"
```

Comment:
I think message gateway is not currently part of ephemeral, please check that once